### PR TITLE
[Needs Review]b07902089 李智源

### PR DESCRIPTION
**Pull request by ZG926**
## What is the most challenging part in this homework?
The most challenging part in this homework is learning how to  use github especially rebasing. 

## Is this homework helpful for you?
Yes,I learned some knowledge about Git and Python

## Is there any suggestion?
-
<br />

### CODE REVIEW

File: src/students/b07902089.py [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+    import utils
+    import io
+    from PIL import Image
+    with request.urlopen(img_url) as response:
+        r = response.read()
+    result_img = Image.open(io.BytesIO(r))
+    result_img = utils.draw_text(result_img, 'b07902089')
+    # Display the image:
+    # result_img.show()
+    result_img.show()
```
Comment (amjltc295) [inline src/students/b07902089.py]: Please comment this line.

**Comment by amjltc295** [comment]
@ZG926  Your commit message is not linked to your profile. See https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/ and https://www.git-tower.com/learn/git/faq/change-author-name-email to fix it.

**Comment by ZG926** [comment]
I have just changed the author of the commit messages to my profile

**Comment by ZG926** [comment]
I've just linked all the previous commit messages to my profile

**Comment by ZG926** [comment]
The code for my task8 is refer to this website :  https://docs.python.org/3.4/howto/urllib2.html

**Pull request merged**